### DISCUSSION TITLE
Changing description of the delete all difficulties dialog

### DISCUSSION THREAD

**Discussion by Alib234** [ppy/osu#29799] [Ideas] [answered]
as the image below shows, it's currently "Artist - Title" and nothing else
I feel like this should be changed to something like "Deleting all difficulties of Artist - Title", "Deleting map: Artist - Title" or maybe something more akin to the other deletion dialogs in lazer like "Are you sure you want to delete Artist - Title?"
![image](https://github.com/user-attachments/assets/9d73bd71-8e8d-4f02-9b7f-8b5201f726bc)

**Comment by peppy** [comment, marked as answer]
https://github.com/ppy/osu/issues/30071
